lib/fluent/plugin/filter_stdout.rb

```diff
@@ -31,7 +30,7 @@ class StdoutFilter < Filter
     def configure(conf)
       super
 
-      @formatter = Plugin.new_formatter(@format)
+      @formatter = Fluent::Plugin.new_formatter(@format)
```

Comment:
This plugin should use `:formatter` plugin helper to instantiate/configure Formatter instance.
But currently, there are no way to keep compatibility of configurations between styles of v0.12 (flat parameters of filter and formatter) and v0.14 (`<format>` subsection).
I'll add that method and document. Please wait for a while.